collage/collage_image.py

```diff
@@ -0,0 +1,52 @@
+from PIL import ImageTk, Image
+from collage import CollageItem, SelectionMixin
+
+
+class PILCollageImage(CollageItem, SelectionMixin):
```

Comment:
Maybe, it would be better to rename file to CollageImage.py for consistency?

Reply:
yes, good idea